src/convert.rs

```diff
@@ -0,0 +1,53 @@
+use bdk::bitcoin::consensus::Decodable;
+use bitcoin::consensus::Encodable;
+
+pub fn bdk_block_hash_to_bitcoin_block_hash(hash: bdk::bitcoin::BlockHash) -> bitcoin::BlockHash {
+    use bdk::bitcoin::hashes::Hash as _;
+    let bytes = hash.as_byte_array().to_vec();
+
+    use bitcoin::hashes::sha256d::Hash;
+    use bitcoin::hashes::Hash as _;
+    let hash: bitcoin::hashes::sha256d::Hash = Hash::from_slice(&bytes).unwrap();
+
+    bitcoin::BlockHash::from_raw_hash(hash)
+}
+
+pub fn bitcoin_tx_to_bdk_tx(
+    tx: bitcoin::Transaction,
+) -> Result<bdk::bitcoin::Transaction, bdk::bitcoin::consensus::encode::Error> {
+    let mut tx_bytes = vec![];
+    tx.consensus_encode(&mut tx_bytes)
+        // Not really going to happen, mostly to satisfy the compiler.
+        .map_err(|err| {
```

Comment:
You can avoid this by using https://docs.rs/bitcoin/latest/bitcoin/consensus/fn.serialize.html, which does not return a `Result<_, _>`.
ie.
```rust
let mut tx_bytes = bitcoin::consensus::serialize(tx);
...
```